test/metabase/db/liquibase_test.clj

```diff
@@ -64,10 +64,11 @@
          (remove str/blank?))))
 
 (deftest consolidate-liquibase-changesets-test
-  (mt/test-drivers #{:h2 :mysql :postgres}
-    (mt/with-temp-empty-app-db [conn driver/*driver*]
-      ;; fake a db where we ran all the migrations, including the legacy ones
-      (with-redefs [liquibase/decide-liquibase-file (fn [& _args] @#'liquibase/changelog-legacy-file)]
+  (with-bindings {#'liquibase/-test-only-*return-caching-parser* false}
```

Comment:
It would be nice to hide this implementation detail with our own macro, something like `mt.liquibase/without-changeset-caching`, where `mt` is shorthand for `metabase.test`.